src/pages/About/MissionAndPurpose/MissionAndPurpose.scss

```diff
@@ -0,0 +1,47 @@
+.title{
```

Comment:
update classNames to new naming conventions above above, like about-mission-purpose__title. I'll send you a good link on how to do this in SCSS via slack. 

Try to keep your CSS to what's necessary rather than the whole figma CSS. Make it Sassy!